build/targets/standard-lifecycle.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
```

Comment:
This gets included by `makefile.proj` which has the DefaultTarget. Would we need one here?